### ISSUE TITLE
window10 cmd 执行docker-compose up -d nginx mysql 

### ISSUE THREAD

**Issue by zhangyang34** [laradock/laradock#3063]
unexpected character "-" in variable name near "-S\nWEAVER_MAX_WORKERS=10\nWEAVER_MAX_CONVERSION_QUEUE=50\nWEAVER_WORKER_TIMEOUT=90\nWEAVER_CONVERSION_FALLBACK=false\n\n# CloudConvert\n# API key can be found: https://cloudconvert.com/use
r/profile\n# CLOUDCONVERT_API=https://api.cloudconvert.com\n# CLOUDCONVERT_KEY=\n\n# Monitoring\n# STATSD_ADDRESS=\n# STATSD_PREFIX=\n# SENTRY_DSN="

**Comment by abelschooleman** [comment]
I have the same problem on linux kernel 5.14.6-arch1-1. This started when docker-compose was upgraded to 2.0.1.
I'm still running laradock with docker-compose 1.29.2 on another machine without problems.

**Comment by davidjr82** [comment]
Same issue in windows after docker upgrade. Workaround to fix it: comment (or delete) the weaver section in docker-compose.yml

**Comment by dhtmdgkr123** [comment]
![image](https://user-images.githubusercontent.com/27611405/135814963-75a1bd3d-c23d-4be0-af4c-021818126ba3.png)
add double queotation and it will work again

**Comment by dhtmdgkr123** [comment]
I have same issue in mac os
![image](https://user-images.githubusercontent.com/27611405/135815635-8b2c4e16-b798-4afb-a0c7-c31c9d3fe6be.png)

in weaver/conf/sample.env  file
add double quotation
![image](https://user-images.githubusercontent.com/27611405/135815318-9f6fe396-d118-441b-a493-2c16f9caf250.png)
and it will work again!

**Comment by garbinmarcelo** [comment]
I got the same error when trying to up apache2 php-worker postgres

I believe this bug is of high priority as it is not possible to initialize the environment. More attention to this issue would be necessary.

**Comment by garbinmarcelo** [comment]
One question I'm in doubt, have the prefixes for container names changed (underline replaced by dash)?

Previous example: website_apache2_1
Current: website-apache2-1

**Comment by dhtmdgkr123** [comment]
@garbinmarcelo yup u can change underline to dash
see [document](https://docs.docker.com/compose/compose-file/compose-file-v3/#container_name)
but if u change underline to dash you can not use [docker-compose scale](https://docs.docker.com/compose/reference/scale/)

**Comment by garbinmarcelo** [comment]
> @garbinmarcelo yup u can change underline to dash see this [document](https://docs.docker.com/compose/compose-file/compose-file-v3/#container_name) also if u change underline to dash you can not use [docker-compose scale](https://docs.docker.com/compose/reference/scale/)

Right, I didn't understand the documentation very well. I don't know if there was any update to the docker that changed the name. 
Because I always used it with underline and now I went up laradock in an old project and saw that everything changed to dash (I tested it on one that already had laradock and now also has a dash.) . 
That's why I was in doubt if it was the laradock or the docker, apparently it was the docker anyway, but I didn't find the option to set underline as default.  :confused:
